test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -151,6 +154,24 @@
         StreamingInboundHandler.trackInboundHandlers();
     }
 
+    @Override
+    public boolean isLogsSupported()
```

Comment:
Nit: maybe `logsSupported` ? I would say `areLogsSupported` but that also doesn't sound very good.

Reply:
was mostly going off java convention, but we don't always follow it in C*, so maybe I should follow the feature enabled convention? 

I do find your methods better English though =D.

Reply:
IMO, prefixing with `is`, `should`, `can`, etc. is more expressive and reads better. 
The `Logs` here is actually log inspection feature. 
Maybe the full form is `isLogInspectionSupported`, or since it is a feature can or cannot be supported, it could also be `canInspectLogs`.

Reply:
According to https://issues.apache.org/jira/browse/CASSANDRA-15234 the pattern is

yaml:
```
<feature>_enabled
```

with getter, so `get<feature>Enabled()`